magnum_cluster_api/resources.py

```diff
@@ -295,7 +297,7 @@ def get_object(self) -> pykube.ConfigMap:
                         "annotations": {
                             "storageclass.kubernetes.io/is-default-class": "true"
                         }
-                        if vt.name == "__DEFAULT__"
+                        if default_volume_type and default_volume_type.name == vt.name
```

Comment:
is it possible that `default_volume_type` equals `None` ?

Reply:
The default_volume_type configuration option is required to have a value from the train version.
So this condition is meaningless.